packages/ocaml-sat-solvers/ocaml-sat-solvers.0.3/descr

```diff
@@ -0,0 +1,5 @@
+This library contains an abstraction layer for integrating SAT
```

Comment:
Apologies, just one last change request: the first line forms the short summary that you see when doing `opam list`, and this `descr` overflows. Do you mind shortening it, for example: "an abstraction layer for integrating SAT solvers into OCaml" would be fine.